pkg/registry/apis/provisioning/jobs/exporter.go

```diff
@@ -177,30 +180,43 @@ func (r *exporter) Export(ctx context.Context,
 		}
 
 		delete(item.Object, "metadata")
-		marshalledBody, err := yaml.Marshal(item.Object)
```

Comment:
we can consider yaml later.. but given that all our tools work with json that seems like the most reasonable choice